### ISSUE TITLE
Validating phone number

### ISSUE THREAD

**Issue by gesseekur** [alexschimpf/Snkrs-Bot#40]
I was wondering if anyone has ran into the issue of validating a phone number after adding to cart - not sure if was a one time thing or I have to account for it now every time. Thanks!

**Comment by alfredog1976** [comment]
Yes I have found in my testing if you attempt to buy a SNKRs release from another location (signed in/purchase location different from last purchase attempt or no verified phone number), it will prompt you to verify phone number before checkout. Also, you can verify your phone number in your Account settings ahead of time.

**Comment by gesseekur** [comment]
Oh okay, great. Just making sure. Thank you
